docs/src/main/asciidoc/mongodb.adoc

```diff
@@ -596,6 +596,12 @@ This behavior must first be enabled by setting the `quarkus.mongodb.metrics.enab
 So when you access the `/q/metrics` endpoint of your application you will have information about the connection pool status.
 When using link:microprofile-metrics[SmallRye Metrics], connection pool metrics will be available under the `vendor` scope.
 
+== Tracing
+
+If you are using the `quarkus-smallrye-opentracing` extension, `quarkus-mongodb-client` can register traces about the commands executed.
+This behavior must first be enabled by setting the `quarkus.mongodb.tracing.enabled` property to `true` in your `application.properties`.
```

Comment:
You also need to add the opentracing mongo client.

To avoid duplicating configuration, better add a link to the related section on the OpenTracing guide

Reply:
Added the info about the requirement and the link to the OpenTracing guide specific section.